spec/features/letter_automation/syncing_spec.rb

```diff
@@ -47,7 +47,8 @@
     it 'will sync the case priority and send the letter automatically' do
       when_the_sync_runs(document_count_changes_by: 1, case_priority_count_changes_by: 1)
       then_a_document_is_queued
-      then_the_case_priority_is(:no_action)
+      then_the_case_priority_is_paused_until_tomorrow
+      # then_the_case_priority_is(:no_action)
```

Comment:
I think actually showing that case is still in the send letter one stage and this is why it's paused is important.

```suggestion
      then_the_case_priority_is(:send_letter_one)
```